lib/crypto/tasks.py

```diff
@@ -318,3 +343,81 @@ def file_supports_firefox(version):
                 fail_silently=True,
                 headers={'Reply-To': 'amo-editors@mozilla.org'})
             addons_emailed.append(addon.pk)
+
+
+@task
+def resign_files(file_ids, **kw):
+    """Used to force re-signing the files.
+
+    This is used in the 'force_resign_files' management command. We need to
+    re-sign the files that were signed with the wrong certificate (see bug
+    1225036).
+
+    It also bumps the version number of the file and the Version, so the
+    Firefox extension update mecanism picks this new signed version and
+    installs it.
+    """
+    log.info(u'[{0}] Signing files.'.format(len(file_ids)))
```

Comment:
This is historic, it's the format used for other management commands. It might be used to grep logs or something, not sure.